isis/CMakeLists.txt

```diff
@@ -40,8 +40,8 @@ set(PACKAGE_VERSION    ${VERSION})
 set(PACKAGE_STRING     "${PACKAGE_NAME} ${VERSION}")
 
 # Other release information
-set(VERSION_DATE              "2018-10-26")
-set(RELEASE_STAGE             "beta") # (alpha, beta, stable)
```

Comment:
Should probably rebase this? I'm guessing it's already in the system but I could be wrong

Reply:
Oops, yes it is. Thanks!

Reply:
My attempt at fixing this did not work. Is there any chance you can tell what I did wrong?